corehq/apps/accounting/payment_handlers.py

```diff
@@ -335,28 +353,40 @@ def create_charge(self, amount, card=None, customer=None):
         )
 
     def update_credits(self, payment_record):
-        self.credit_line = CreditLine.add_credit(
-            payment_record.amount, account=self.account, subscription=self.subscription,
-            product_type=self.product_type, feature_type=self.feature_type,
-            payment_record=payment_record,
-        )
+        for feature in self.features:
+            feature_amount = feature['amount']
+            if feature_amount >= 0.5:
+                self.credit_lines.append(CreditLine.add_credit(
+                    feature_amount,
+                    account=self.account,
+                    subscription=self.subscription,
+                    feature_type=feature['type'],
+                    payment_record=payment_record,
+                ))
+        for product in self.products:
+            plan_amount = product['amount']
+            if plan_amount >= 0.5:
+                self.credit_lines.append(CreditLine.add_credit(
+                    plan_amount,
+                    account=self.account,
+                    subscription=self.subscription,
+                    product_type=product['type'],
+                    payment_record=payment_record,
+                ))
 
     def process_request(self, request):
         response = super(CreditStripePaymentHandler, self).process_request(request)
-        if hasattr(self, 'credit_line'):
+        if hasattr(self, 'credit_lines'):
```

Comment:
In what situation would this attribute not be present? Looks like it's [set in `__init__`](https://github.com/dimagi/commcare-hq/blob/credit-card-prepay/corehq/apps/accounting/payment_handlers.py#L320). Would it make sense to change this to
```py
if self.credit_lines:
```